### PR TITLE
Tile: Long description in stack tile is truncated

### PR DESCRIPTION
**Pull request by Bazif-Khan**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description

Fixes: https://github.com/appsody/website/issues/190

### CODE REVIEW

**Comment by tomleah** [comment]
I think we need a more versatile solution here as the problem remains with an even longer text description. 

Maybe we have the description go into `...` or we limit the length of descriptions with the stack linter. I would rather not limit stacks because of the website however.

**Comment by neeraj-laad** [comment]
I agree we might need a longer text, but perhaps we definitely need stack providers to give us a concise shortDescription that we show on website and perhaps on cli. As having arbitrary long desc could cause other sorts of problem and might be best if that goes on README etc.

@Bazif-Khan a screen shot would help with something like this. 

I think if that looks ok we can merge this PR but keep the issue open as we need a better solution for it long term.

**Comment by tomleah** [comment]
I used "Spring Boot on Open Liberty & OpenJ9 using Maven Spring Boot on Open Liberty & OpenJ9 using Maven" as the description and it then stretches to 4 lines instead of max of 3. Could you reduce the character limit

File: src/components/tile.js [hunk 1]
```diff
@@ -3,6 +3,13 @@ import React from "react";
 class Tile extends React.Component {
   constructor(props) {
     super(props);
+    if(this.props.desc.length > 75) {
```
Comment (tomleah) [inline src/components/tile.js]: Could maybe move 75 into a variable

File: src/styles/main.css [hunk 2]
```diff
@@ -175,13 +175,21 @@ section.landing-section h2 {
...
+}
 .tile a.btn {
   position: absolute;
-  bottom: 2em;
+  bottom: 1.5em;
   left: 0;
   right: 0;
 }
 
+.tile p {
```
Comment (tomleah) [inline src/styles/main.css]: Do we need this now we using the different truncating method? Or is this needed for style still?

Comment (Bazif-Khan) [reply src/styles/main.css]: It is being used to fit the text between button and tile heading.

**Comment by Bazif-Khan** [comment]
> I used "Spring Boot on Open Liberty & OpenJ9 using Maven Spring Boot on Open Liberty & OpenJ9 using Maven" as the description and it then stretches to 4 lines instead of max of 3. Could you reduce the character limit

I have reduced the character limit.

**Pull request merged**